x-pack/plugins/enterprise_search/public/applications/workplace_search/views/overview/workplace_search_gate.tsx

```diff
@@ -0,0 +1,751 @@
+              <EuiFlexItem grow={false}>
+                <EuiButton href={feature.actionLink} iconType="sortRight" iconSide="right">
+                  {feature.actionLabel}
+                </EuiButton>
+              </EuiFlexItem>
+            )}
+
+            <EuiFlexItem grow={false}>
+              <EuiLink href={feature.learnMore} target="_blank">
+                {i18n.translate(
+                  'xpack.enterpriseSearch.workplaceSearch.gateForm.educationalPanel.learnMore',
+                  {
+                    defaultMessage: 'Learn More',
+                  }
+                )}
+              </EuiLink>
+            </EuiFlexItem>
+
+            {feature.addOnLearnMoreLabel !== undefined && feature.addOnLearnMoreUrl !== undefined && (
+              <EuiFlexItem grow={false}>
+                <EuiLink type="button" href={feature.addOnLearnMoreUrl} target="_blank" external>
+                  <EuiSpacer />
+                  {feature.addOnLearnMoreLabel}
+                </EuiLink>
+              </EuiFlexItem>
+            )}
+          </EuiFlexGroup>
+        </EuiCallOut>
+      </EuiPanel>
+    );
+  } else {
+    return (
+      <>
+        <EuiSpacer />
+        <EuiFormRow
+          label={i18n.translate(
+            'xpack.enterpriseSearch.workplaceSearch.gateForm.featureOther.Label',
+            {
+              defaultMessage: "Can you explain what other feature(s) you're looking for?",
+            }
+          )}
+        >
+          <EuiTextArea
+            onChange={(e) => {
+              setFeaturesOther(e.target.value);
+            }}
+          />
+        </EuiFormRow>
+      </>
+    );
+  }
+};
+export const WorkplaceSearchGate: React.FC<{ isLoading: boolean }> = ({ isLoading }) => {
+  const options = [
+    {
+      dropdownDisplay: (
+        <Fragment>
+          <strong>
+            {i18n.translate(
+              'xpack.enterpriseSearch.workplaceSearch.gateForm.superSelect.contentSource.title',
+              {
+                defaultMessage: 'Content sources',
+              }
+            )}
+          </strong>
+          <EuiText size="s" color="subdued">
+            <p>
+              {i18n.translate(
+                'xpack.enterpriseSearch.workplaceSearch.gateForm.superSelect.contentSource.description',
+                {
+                  defaultMessage:
+                    'Extract the content of synced source files to make them searchable',
+                }
+              )}
+            </p>
+          </EuiText>
+        </Fragment>
+      ),
+      inputDisplay: i18n.translate(
+        'xpack.enterpriseSearch.workplaceSearch.gateForm.superSelect.contentSource.inputDisplay',
+        {
+          defaultMessage: 'Content ources',
+        }
+      ),
+      value: i18n.translate(
+        'xpack.enterpriseSearch.workplaceSearch.gateForm.superSelect.value.contentSource.value',
+        {
+          defaultMessage: 'contentSources',
```

Comment:
> This and the other value props should not be translated, they're variables/enums.

So all the labels in would be translated but not the value ?

Reply:
Yes. We translate things that are shown to the user, for internationalization purposes. But the `value` is not shown to the user in any way, it is an internal variable. If you translate that, the logic will break, because we're doing string checks against different, untranslated ids.